File: Python/positive_negative.py
# Python program to check whether a number is positive or negative or zero.

num = int(input("Enter a number: "))

if num > 0:
   print('Entered number is a positive number.')
elif num == 0:
   print('Entered number is zero.')
else:
   print('Entered number is a negative number.')

# Printing the result using f string or string literals
# print(f'Entered number is a postive number') or print(f'{num} is a positive number')

File: C/positive_negative.c
// C program to check whether a number is positive or negative or zero.

#include <stdio.h>

int main() {

    int num;
    printf("Enter a number: ");
    scanf("%d", &num);
    if (num <= 0) {
        if (num == 0)
            printf("The number is equals to 0");
        else
            printf("You entered %d and it is a negative number", num);
    } 
    else
        printf("You entered %d and it is a positive number", num);

    return 0;
}